cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,29 @@
+import { generateUser } from '../support/generate';
+
+Cypress.on('uncaught:exception', () => {
+  return false;
+});
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const { email, username, password } = generateUser();
+
+    cy.visit('https://react-redux.realworld.io/#/register?_k=spysa9');
```

Comment:
Hardcoding URLs in the tests is not a good practice. If the URL changes in the future, you will have to change it in every place it's used. Consider storing the base URL in a configuration file or environment variable and just append the path specific to the test.